### DISCUSSION TITLE
depends routers by True/False function

### DISCUSSION THREAD

**Discussion by galos147** [fastapi/fastapi#6106] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from pydantic import BaseModel
from fastapi import FastAPI

app = FastAPI()

def permission(token):
    username = token['username']
    group = 'test'
    apirequest = "calling api check if user exists in group"
    if apirequest:
        return True
    else:
        return False

@app.get("/")
def read_root():
    return {"Hello": "World"}
```

### Description

hello,
i have a function that check if user exists in AD-Group by getting the username from token.
if user exist in group the answer will be True else False.
i want to create a depends on routers that will running only if answer is - True

Can you help me please 
thank you

### Operating System

Windows

### FastAPI Version

0.88.0

### Python Version

3.8

**Comment by jgould22** [comment, marked as answer]
Something like this 
```python
def permission(token):
    username = token['username']
    group = 'test'
    apirequest = "calling api check if user exists in group"
    if not apirequest:
        raise HTTPException(status_code=403, detail="Forbidden")
```

**Comment by galos147** [comment]
Hi I was able to accomplish this using dual depends.
